### DISCUSSION TITLE
Custom CSS Class

### DISCUSSION THREAD

**Discussion by elise-lcdb** [inkline/inkline#326] [Need help]
Hello!

Is it possible to add a custom class to some of the components such as the Input?

When I tried adding a class directly onto the component, I noticed that it doesn't apply to the parent div of the component, so I can't adjust the border radius for example, or give it a different background color.

<img width="824" alt="CleanShot 2022-10-22 at 22 29 36@2x" src="https://user-images.githubusercontent.com/31904512/197361179-fef54655-84ef-4153-8b8f-7e0e63abf3a9.png">

![CleanShot 2022-10-22 at 22 27 46@2x](https://user-images.githubusercontent.com/31904512/197361158-c563ada7-9d61-4729-9d0e-ff6547202508.png)

**Comment by five1** [comment]
Have you tried using the [_border utility class](https://www.inkline.io/docs/utilities/border#border-radius)?
`<i-input class="_border-radius:5" ..`

You can learn more about utility classes in the [class naming convention documentation](https://www.inkline.io/docs/preface/concepts#class-naming-convention).
